src/App.jsx

```diff
@@ -14,70 +15,88 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [valueGood, SetValueGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {valueGood ? (
+          <>
+            {`${valueGood} is selected`}
             <button
-              data-cy="AddButton"
+              onClick={() => SetValueGood('')}
+              data-cy="ClearButton"
               type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+              className="delete ml-3"
+            />
+          </>
+        ) : (
+          'No goods selected'
+        )}
+      </h1>
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <>
+              {valueGood === good ? (
+                <tr data-cy="Good" className="has-background-success-light">
+                  <td>
+                    <button
+                      onClick={() => SetValueGood('')}
+                      data-cy="RemoveButton"
+                      type="button"
+                      className="button is-info"
+                    >
+                      -
+                    </button>
+                  </td>
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+                  <td data-cy="GoodTitle" className="is-vcentered">
+                    {good}
+                  </td>
+                </tr>
+              ) : (
+                <tr data-cy="Good">
+                  <td>
+                    <button
+                      onClick={() => {
+                        SetValueGood(good);
+                      }}
+                      data-cy="AddButton"
+                      type="button"
+                      className="button"
+                    >
+                      +
+                    </button>
+                  </td>
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+                  <td data-cy="GoodTitle" className="is-vcentered">
+                    {good}
+                  </td>
+                </tr>
```

Comment:
you write this part twice

```
<td data-cy="GoodTitle" className="is-vcentered"> 
  {good}
</td>
```

try to change the logic in order not to do that

Reply:
Please, re-check this comments and try to implement this. If you have any questions, ask them in the chat or come to the QnA session
![image](https://github.com/mate-academy/react_goods-selector-js/assets/54936044/9784950f-e366-4a86-a793-a126f8082a1a)